src/ErrorBoundary.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+interface ErrorBoundaryState {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends React.Component<object, ErrorBoundaryState> {
+  constructor(props: object) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
+
+  componentDidCatch(error: Error, info: React.ErrorInfo) {
+    console.log(error, info.componentStack);
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <div className="flex justify-center items-center h-screen">
+          <h1 className="text-6xl text-red-600">Something went wrong.</h1>
+        </div>
+      );
+    }
+
+    return (this.props as React.PropsWithChildren<object>).children;
```

Comment:
I think you can return `this.prop.children`?

Reply:
In this case I have a type script error: Property 'children' does not exist on type 'object'. I found only this way to resolve it.